iac/terraform/aws/auth_cognito/cognito.tf

```diff
@@ -0,0 +1,69 @@
+resource "aws_cognito_user_pool" "user_pool" {
+  name = var.user_pool_name
+
+  schema {
+    attribute_data_type = "String"
+    name                = "email"
+    required            = true
+    mutable             = false
+  }
+
+  schema {
+    attribute_data_type = "String"
+    name                = "name"
+    required            = true
+    mutable             = true
+  }
+
+  schema {
+    attribute_data_type = "String"
+    name                = "phone_number"
+    required            = true
+    mutable             = true
+  }
+
+  auto_verified_attributes = ["email"]
+  username_attributes      = ["email"]
+
+  lambda_config {
+    custom_message    = module.lambda["custom_email_template"].lambda_arn
+    post_confirmation = module.lambda["post_confirmation"].lambda_arn
+  }
+
+  password_policy {
+    minimum_length    = 10
```

Comment:
Minimum length 6 is better I believe. A tf var can be used.

Reply:
I think regarding the minimum length of password `Khirul Vai` talked with you. And Inted of hardcoded password length now used a variable. And you can find this in this following [commit](https://github.com/CraftsmenLtd/BloodConnect/pull/113/commits/337254530d2413a8fe5435148b1f672b814a34ca)